core/src/main/java/com/amplifyframework/core/Amplify.java

```diff
@@ -84,9 +89,9 @@ private Amplify() {
         throw new UnsupportedOperationException("No instances allowed.");
     }
 
-    // The fact that this is a LinkedHashMap, and not a Map, is important.
+    // The fact that this is a LinkedHashMap is important.
```

Comment:
!!!!!

Reply:
Agreed!  How about this:

 - Keep `CATEGORIES` as a private `LinkedHashMap`, and the `Amplify` class can continue using it as is.
 - Create a public getter method `getCategoriesMap()`, which returns an immutable, unordered categories `Map`, which is used by the DevMenu.